src/sentry/static/sentry/app/components/charts/panelChart/index.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {Panel, PanelHeader} from 'app/components/panels';
+
+import Legend from './legend';
+
+const PanelChart = styled(
+  class extends React.Component {
+    static propTypes = {
+      showLegend: PropTypes.bool,
+      title: PropTypes.node,
+      ...Legend.propTypes,
+    };
+
+    static defaultProps = {
+      showLegend: true,
+    };
+
+    constructor(props) {
+      super(props);
+    }
+
+    render() {
+      // Don't destructure `height` so that we can pass to children
```

Comment:
I didn't quite get this since I don't see `height` in the proptypes of both this component or `Legend`.